src/TextInput/TextInput.features.stories.tsx

```diff
@@ -110,25 +110,40 @@ export const WithTrailingIcon = () => (
 )
 
 export const WithTrailingAction = () => {
-  const [value, setValue] = useState('')
+  const [value, setValue] = useState('Sample Text')
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValue(event.target.value)
   }
+
   return (
     <Box as="form">
       <FormControl>
         <FormControl.Label>Default label</FormControl.Label>
         <TextInput
           trailingAction={
-            <TextInput.Action
-              onClick={() => {
-                setValue('')
+            <Box
+              sx={{
+                width: '30px',
+                display: 'flex',
+                alignItems: 'center',
```

Comment:
I would have to come back to you next week to review this approach!

Reply:
yeah it says the screenshot comparison fails. 

<img width="1032" alt="Screenshot 2023-11-14 at 9 19 40 AM" src="https://github.com/primer/react/assets/68850280/61f9cec6-9ab6-497c-ae1d-ae3924904afe">

anyway to view the difference or update the existing snapshots accordingly (the way i did the test update for AvatarStack issue)

Reply:
That's expected because we did make a visual change :)

Added the label to update snapshots, should update in a few minutes

Reply:
it seems update snapshots label has been removed, anyway to achieve this?

Reply:
I can add that for you